src/coveralls.jl

```diff
@@ -102,8 +102,14 @@ module Coveralls
                 data["git"]["branch"] = split(ENV["GIT_BRANCH"], "/")[2]
             end
         elseif haskey(ENV, "GITHUB_ACTION")
+            data["service_job_id"] = ENV["GITHUB_RUN_ID"]
             data["service_name"] = "github"
             data["git"] = parse_git_info(git_info)
+
+            event_path = open(JSON.Parser.parse, ENV["GITHUB_EVENT_PATH"])
+            github_pr_info = get(event_path, "pull_request", Dict())
+            github_pr = get(github_pr_info, "number", "")
+            isempty(github_pr) || (data["service_pull_request"] = github_pr)
```

Comment:
Ooooh, that's nice, I wasn't aware of it, else I would have suggested it for PR #288 -- I wonder if perhaps you or @kyungminlee would be interested in adjusting the Codecov code path to also parse the `GITHUB_EVENT_PATH` JSON. In general, we try to keep the Codecov and Coveralls support in sync (but I merged PR #288 anyway, because (a) I forgot 😨 and (b) honestly, I'd rather have something that works in now than to not have it at all because it's perfect...

Anyway: very nice 😄

Reply:
Sure I can do that.

Reply:
It's a bit tricky to use in general though, because for push builds you don't have that much info (like I could not find how to e.g. extract the branch name in a quick search).

Reply:
I guess one then needs to combine techniques from PR #288 and this one... Ah well.